docs/SPIRVRepresentationInLLVM.rst

```diff
@@ -483,6 +483,13 @@ extra operands ``i32 1`` and ``i32 2``.
 decorates the argument ``b`` of ``k`` with ``Restrict`` in SPIR-V while not
 adding any decoration to argument ``a``.
 
+Loop controls and loop metadata
+-------------------------------
+
+SPIR-V Loop controls that do not have a corresponding community LLVM metadata
```

Comment:
What does "community" mean in this context? I think just "LLVM metadata" should be good enough to describe the intention.

Reply:
@bader the intention is to distinguish between vendor-specific metadata and metadata that is present in community-LLVM

Reply:
I think "community" is ambiguous, it could refer to any community.  What about the following?
```suggestion
SPIR-V Loop controls that do not have corresponding `llvm.loop` metadata
```

Reply:
@svenvh I have updated the text according to your suggestion.